src/CMakeLists.txt

```
@@ -86,24 +86,23 @@ set(_SOURCES
   )
 
 
-# create library with .cpp, .cu and .f90 sources
-add_library(sirius "${_SOURCES};${_CUSOURCES};${_FSOURCES}")
-
-target_compile_features(sirius PUBLIC cxx_std_14)
-target_compile_features(sirius PUBLIC c_std_99)
-target_compile_features(sirius PUBLIC cuda_std_14)
-set_target_properties(sirius PROPERTIES POSITION_INDEPENDENT_CODE ON)
+  # create library with .cpp, .cu and .f90 sources
+add_library(sirius_cxx "${_SOURCES};${_CUSOURCES};")
+target_compile_features(sirius_cxx PUBLIC cxx_std_14)
```

```suggestion
target_compile_features(sirius_cxx PUBLIC cxx_std_17)
```
Isn't SIRIUS using C++17 now?